pkg/controller/clusterapi_test.go

```diff
@@ -4,6 +4,7 @@ import (
 	"context"
 	"testing"
 
+	etcdv1 "github.com/mrajashree/etcdadm-controller/api/v1beta1"
```

Comment:
Can you not reference github.com/aws/etcdadm-controller instead of mrajashree fork?

Reply:
There's a dependency in go.mod on the mrajashree fork, so changing to the aws fork will require a bigger change which I'm not inclined to do here.